egs/mls_english/ASR/prepare.sh

```
@@ -0,0 +1,143 @@
+	if [ ! -f data/lang/transcript.txt ]; then
+		log "Generating transcripts for BPE training"
+		python local/utils/generate_transcript.py \
+			--dataset-path $dl_dir/mls_english \
+			--lang-dir data/lang \
+			--split train 
+	fi
+fi
+
+if [ $stage -le 5 ] && [ $stop_stage -ge 5 ]; then
+	log "Stage 5: Prepare BPE tokenizer"
+	for vocab_size in ${vocab_sizes[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)